core/dbt/contracts/graph/manifest.py

```diff
@@ -1232,6 +1257,8 @@ def __post_serialize__(self, dct):
         for unique_id, node in dct["nodes"].items():
             if "config_call_dict" in node:
                 del node["config_call_dict"]
+            if "state_relation" in node:
+                del node["state_relation"]
```

Comment:
It doesn't feel like `state_relation` is a thing we should need/want to include in serialized `manifest.json`. It's really just for our internal use.